addons/mail/models/models.py

```diff
@@ -163,10 +163,9 @@ def _alias_get_error_message(self, message, message_dict, alias):
         if alias.alias_contact == 'followers':
             if not self.ids:
                 return _('incorrectly configured alias (unknown reference record)')
-            if not hasattr(self, "message_partner_ids") or not hasattr(self, "message_channel_ids"):
+            if not hasattr(self, "message_partner_ids"):
                 return _('incorrectly configured alias')
```

Comment:
wouldn't it be nice to tell what is incorrectly configured?

Reply:
Would be. But as it is part of something sent by email, honestly, I will keep it like that. Already too much unrelated work in this PR.